report_form_saver.py

```diff
@@ -193,33 +194,60 @@ def validate_email(self, email):
             flash(f"Email ({email}) is not valid. Please enter a valid email address.")
             return False
 
+    @staticmethod
+    def make_tz_aware(time):
+        default_tz = pendulum.timezone(conf.get("core", "default_timezone"))
+        time_of_day_to_local = pendulum.datetime(
+            1970,
+            1,
+            1,
+            time.hour,
+            time.minute,
+            tzinfo=default_tz
+        )
+        return time_of_day_to_local
+
+    def convert_to_default_timezone(self, time):
+        default_tz = pendulum.timezone(conf.get("core", "default_timezone"))
```

Comment:
Same question, why are these focused only on times?

Reply:
this is just for the backend (converting everything to default tz in the variable and initially on the form before conversion). all the stuff converting to the local timezone is on the frontend (javascript) because that's where the local timezone is stored by airflow

Reply:
Documentation will help here.